requirements.txt

```diff
@@ -1,4 +1,5 @@
 traitlets
 nbconvert==4.1
 scons
```

Comment:
If we're going to be having multiple build systems supported, I would actually not make any of them a dependency -- so rather than adding doit as a dependency, remove scons as a dependency.